pkgs/servers/matrix-synapse/default.nix

```diff
@@ -1,6 +1,6 @@
-{ stdenv, pythonPackages, fetchurl, fetchFromGitHub }:
+{ lib, pkgs, stdenv, python2Packages, fetchurl, fetchFromGitHub, fetchpatch }:
```

Comment:
Don't pass in `pkgs`. Unused as well.

Reply:
Sorry, I must have messed something up when rebasing.